presto-orc/src/test/java/io/prestosql/orc/TestTupleDomainOrcPredicate.java

```diff
@@ -88,7 +88,7 @@ private static ColumnStatistics booleanColumnStats(Long numberOfValues, Long tru
         if (trueValueCount != null) {
             booleanStatistics = new BooleanStatistics(trueValueCount);
         }
-        return new ColumnStatistics(numberOfValues, 2L, booleanStatistics, null, null, null, null, null, null, null);
+        return new ColumnStatistics(numberOfValues, 2L, booleanStatistics, null, null, null, null, null, null, null, null);
```

Comment:
I expected to see a test for the new timestamp stats here.  Is that possible?

Reply:
I see this is in the second commit.  In this case I would merge the two commits together to improve readability of the diff.